src/main/resources/db/migration/postgresql/V1_2__ADD_MESSAGE_TYPE_COLUMN.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE IF EXISTS message_log
```

Comment:
Would it also be beneficial to have a column for the last processed attempt time, we currently have `created` and `completed`. Makes sense to have `processed` too?

Reply:
Yes I think it would be beneficial here and for the migration

Reply:
Added